gamestonk_terminal/parent_classes.py

```diff
@@ -116,69 +124,75 @@ def switch(self, an_input: str) -> List[str]:
         # Empty command
         if not an_input:
             console.print("")
-            return self.queue
 
-        # Navigation slash is being used
-        if "/" in an_input:
+        # Navigation slash is being used first split commands
+        elif "/" in an_input:
             actions = an_input.split("/")
 
             # Absolute path is specified
             if not actions[0]:
-                an_input = "home"
-            # Relative path so execute first instruction
-            else:
-                an_input = actions[0]
+                actions[0] = "home"
 
             # Add all instructions to the queue
-            for cmd in actions[1:][::-1]:
+            for cmd in actions[0:][::-1]:
```

Comment:
If it starts at 0, then I don't think we need `[0:]`

Reply:
ha ye, didnt clean properly